mockld/sdk_data.go

```diff
@@ -252,6 +252,11 @@ func (b *ClientSDKDataBuilder) Flag(key string, props ClientSDKFlag) *ClientSDKD
 	return b
 }
 
+func (b *ClientSDKDataBuilder) FullFlag(props ClientSDKFlagWithKey) *ClientSDKDataBuilder {
+	b.flags[props.Key] = props.ClientSDKFlag
+	return b
+}
+
```

Comment:
The client-side data is a little funny because for flag patches, you want an object that includes a `key` property (ClientSDKFlagWithKey), but for the initial data or stream `put` event, you want it without the `key` (ClientSDKFlag). ClientSDKFlagWithKey is more convenient for generating test data since it includes all of the properties in one struct, so I added this method to let us use it everywhere.